exe-unit/components/http-proxy/src/headers.rs

```diff
@@ -0,0 +1,40 @@
+use crate::http_to_gsb::HttpToGsbProxy;
+use actix_http::header::HeaderMap;
+use reqwest::RequestBuilder;
+use std::collections::HashMap;
+
+pub struct Headers {}
+
+impl Headers {
+    const IGNORED_HEADERS: [&'static str; 3] = ["host", "content-length", "connection"];
```

Comment:
You hardcoded headers.
There is always low effort way to make it more flexible. For example you could have field in `Headers` struct for keeping track what to ignore and a function for adding new headers. This could be used like this:
```
let filter = Headers::default().ignore("sth").ignore("sth2");
```
`default` would add default ignores to list.